.evergreen/config.yml

```diff
@@ -1978,6 +2004,22 @@ tasks:
         - func: "run load-balancer"
         - func: "run tests"
 
+    - name: "oidc-auth-test-azure-latest"
+      commands:
+      - command: shell.exec
+        params:
+          shell: bash
+          script: |-
+            set -o errexit
+            ${PREPARE_SHELL}
+            cd src
+            git add .
```

Comment:
Humor me if this is a silly question. Is the `${PREPARE_SHELL}` command responsible for adding in the new files for the commit?

Reply:
No, it sets up common env variables.